### PR TITLE
Change cluster-group-upgrade CI base image to ubi-minimal

### PR DESCRIPTION
**Pull request by sudomakeinstall2**
Signed-off-by: Saeid Askari <saskari@redhat.com>

### CODE REVIEW

**Comment by sudomakeinstall2** [comment]
@jc-rh @nishant-parekh

File: ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 base_images:
   base:
-    name: ubi
+    name: ubi-minimal
```
Comment (nishant-parekh) [inline ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml:3]: @sudomakeinstall2 you need to make this change in other branch config files too..
https://github.com/openshift/release/tree/master/ci-operator/config/openshift-kni/cluster-group-upgrades-operator

**Review by nishant-parekh — COMMENTED** [review]
I think you need to make the change in older config files too.. and then run `make update` to update jobs associated with the config change..

File: ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml (hunk 1 above)
Comment (sudomakeinstall2) [reply ci-operator/config/openshift-kni/cluster-group-upgrades-operator/openshift-kni-cluster-group-upgrades-operator-main.yaml:3]: fixed

**Review by nishant-parekh — APPROVED** [review]

**Comment by nishant-parekh** [comment]
/pj-rehearse pull-ci-openshift-kni-cluster-group-upgrades-operator-main-integration

**Comment by jc-rh** [comment]
/pj-rehearse pull-ci-openshift-kni-cluster-group-upgrades-operator-main-integration

**Pull request merged**